File: yht0827/1월/1월 15일/가장 큰 수.java
public String solution(int[] numbers) {
	String[] stringNumbers = new String[numbers.length];

	for (int i = 0; i < numbers.length; i++) {
		stringNumbers[i] = Integer.toString(numbers[i]);
	}

	Arrays.sort(stringNumbers, new Comparator<String>() {
		@Override
		public int compare(String o1, String o2) {
			return (o2 + o1).compareTo(o1 + o2);
		}
	});

	if (stringNumbers[0].equals("0"))
		return "0";

	return String.join("", stringNumbers);
}

File: zuzubibi/old/1월/1월 15일/가장 큰 수.py
def solution(nums):
    nums = list(map(str, nums))
    nums.sort(key=lambda x : x* 3, reverse=True)
    return str(int(''.join(nums)))

print(solution([6, 10, 2])) # '6210'
print(solution([3, 30, 34, 5, 9])) # "9534330"